apps/wallet-dashboard/components/SupplyIncreaseVestingOverview.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCurrentAccount, useIotaClient } from '@iota/dapp-kit';
+import { useGetSupplyIncreaseVestingObjects } from '@/hooks';
+import {
+    ButtonType,
+    Card,
+    CardAction,
+    CardActionType,
+    CardBody,
+    CardType,
+    LabelText,
+    LabelTextSize,
+    Panel,
+    Title,
+} from '@iota/apps-ui-kit';
+import { StakeDialog, useStakeDialog } from './Dialogs';
+import { TIMELOCK_IOTA_TYPE, useCountdownByTimestamp, useFormatCoin } from '@iota/core';
+import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
+import SvgClock from '@iota/ui-icons/src/Clock';
+import { useQueryClient } from '@tanstack/react-query';
+
+export function SupplyIncreaseVestingOverview() {
+    const account = useCurrentAccount();
+    const address = account?.address || '';
+    const iotaClient = useIotaClient();
+    const queryClient = useQueryClient();
+    const {
+        nextPayout,
+        supplyIncreaseVestingSchedule,
+        supplyIncreaseVestingMapped,
+        supplyIncreaseVestingStakedMapped,
+    } = useGetSupplyIncreaseVestingObjects(address);
+
+    const {
+        isDialogStakeOpen,
+        stakeDialogView,
+        setStakeDialogView,
+        selectedStake,
+        selectedValidator,
+        setSelectedValidator,
+        handleCloseStakeDialog,
+        handleNewStake,
+    } = useStakeDialog();
+
+    const formattedLastPayoutExpirationTime = useCountdownByTimestamp(
+        Number(nextPayout?.expirationTimestampMs),
+        false,
+        false,
+    );
+    const [formattedNextPayout, nextPayoutSymbol, nextPayoutResult] = useFormatCoin(
+        nextPayout?.amount,
+        IOTA_TYPE_ARG,
+    );
+
+    const [formattedAvailableStaking, availableStakingSymbol] = useFormatCoin(
+        supplyIncreaseVestingSchedule.availableStaking,
+        IOTA_TYPE_ARG,
+    );
+
+    const showSupplyIncreaseVestingOverview =
+        supplyIncreaseVestingMapped.length > 0 || supplyIncreaseVestingStakedMapped.length > 0;
+
+    function handleOnSuccess(digest: string): void {
+        iotaClient
+            .waitForTransaction({
+                digest,
+            })
+            .then(() => {
+                queryClient.invalidateQueries({
+                    queryKey: [
+                        'get-timelocked-staked-objects',
+                        'get-all-owned-objects',
+                        account?.address,
+                        {
+                            StructType: TIMELOCK_IOTA_TYPE,
+                        },
+                    ],
+                });
```

Comment:
we should revert this part of the code to invalidate queries separately https://github.com/iotaledger/iota/pull/4420#discussion_r1883989523